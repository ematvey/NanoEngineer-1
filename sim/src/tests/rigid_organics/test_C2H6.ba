LENGTH 0 1 1.52944172821
LENGTH 0 2 1.09576366065
LENGTH 0 3 1.08908080508
LENGTH 0 4 1.09785882517
LENGTH 1 5 1.09576366065
LENGTH 1 6 1.09785882517
LENGTH 1 7 1.08908080508
ANGLE 0 1 5 111.527787199
ANGLE 0 1 6 111.0753273
ANGLE 0 1 7 111.822375733
ANGLE 1 0 2 111.527787199
ANGLE 1 0 3 111.822375733
ANGLE 1 0 4 111.0753273
ANGLE 2 0 3 107.383354096
ANGLE 2 0 4 107.431685603
ANGLE 3 0 4 107.369308986
ANGLE 5 1 6 107.431685603
ANGLE 5 1 7 107.383354096
ANGLE 6 1 7 107.369308986
