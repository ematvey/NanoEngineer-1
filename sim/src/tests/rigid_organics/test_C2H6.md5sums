stdout 7FA4654EF12329EF1B242321F32541A5
stderr D41D8CD98F00B204E9800998ECF8427E
