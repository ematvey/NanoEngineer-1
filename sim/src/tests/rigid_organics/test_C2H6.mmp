mmpformat 050502 required; 050706 preferred
kelvin 300
group (View Data)
info opengroup open = True
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (1.000000, 0.000000, 0.000000, 0.000000) (5.187532) (-0.000000, -0.000000, -0.000000) (1.000000)
egroup (View Data)
group (C2H6)
info opengroup open = True
mol (C2H6.pdb) cpk
atom 1 (6) (308, -220, -337) def
atom 2 (6) (-681, 634, 330) def
bond1 1
atom 3 (1) (-35, -1896, -24) def
bond1 1
atom 4 (1) (590, -637, 778) def
bond1 1
atom 5 (1) (1545, -615, -740) def
bond1 1
atom 6 (1) (55, 1856, -38) def
bond1 2
atom 7 (1) (-1152, 808, 1054) def
bond1 2
atom 8 (1) (-979, 86, -905) def
bond1 2
egroup (C2H6)
end1
group (Clipboard)
info opengroup open = False
egroup (Clipboard)
end molecular machine part C2H6

