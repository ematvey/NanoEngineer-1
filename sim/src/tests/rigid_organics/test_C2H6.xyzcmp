8
C2H6.mmp
C 0.449000 -0.619000 -0.006000
C -0.449000 0.619000 0.006000
H -0.140000 -1.543000 -0.005000
H 1.106000 -0.651000 0.862000
H 1.086000 -0.636000 -0.900000
H 0.140000 1.543000 0.005000
H -1.086000 0.636000 0.900000
H -1.106000 0.651000 -0.862000
