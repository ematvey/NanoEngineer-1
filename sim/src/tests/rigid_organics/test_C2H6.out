======= test_C2H6.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= lengthsangles =======
OK
======= stderr =======
cutover to fine tolerance at 3
linearMinimize allocated 0 instead of 1
minimize allocated 0 instead of 1
== structure comparison ==
cutover to fine tolerance at 1
